ess/src/API/EMBC.ESS/Resources/Suppliers/SupplierRepository.cs

```diff
@@ -207,6 +207,8 @@ private void AddTeamSuppliers(era_supplier existingSupplier, era_supplier update
                 var team = essContext.era_essteams
                         .Where(t => t.era_essteamid == ts1._era_essteamid_value).SingleOrDefault();
 
+                if (team == null) continue;
```

Comment:
should this throw an exception?

Reply:
If you send a null Supplier.Team, without this skip it will still create a team supplier record with a null team. I don't know if there's a case where you'd be deleting all team supplier records for some supplier, but if that's not an allowed business process the manager can catch it.